usr.sbin/inetd/inetd.c

```diff
@@ -2679,7 +2679,9 @@ rl_process(struct servtab *sep, int ctrl)
 		now = rl_time();
 		sep->se_time = now;
 		istimevalid = true;
-	} else if (sep->se_count >= sep->se_service_max) {
+	}
+	
+	if (sep->se_count >= sep->se_service_max) {
```

Comment:
This is a behavior change for inetd. Services that specified 0 for max and expected 1 service to start will no longer start. Services that specified N for max and expected (N-1) services to start will allow one more service than expected.